Menus/SettingsMiscGUI.h

```diff
@@ -0,0 +1,59 @@
+#ifndef _RTESETTINGSMISCGUI_
+#define _RTESETTINGSMISCGUI_
+
+namespace RTE {
+
+	class GUIControlManager;
+	class GUICollectionBox;
+	class GUICheckbox;
+	class GUIEvent;
+
+	/// <summary>
+	/// Handling for misc settings through the game settings user interface.
+	/// </summary>
+	class SettingsMiscGUI {
+
+	public:
+
+#pragma region Creation
+		/// <summary>
+		/// Constructor method used to instantiate a SettingsMiscGUI object in system memory and make it ready for use.
+		/// </summary>
+		/// <param name="parentControlManager">Pointer to the parent GUIControlManager which owns all the GUIControls of this SettingsMiscGUI.</param>
+		explicit SettingsMiscGUI(GUIControlManager *parentControlManager);
+#pragma endregion
+
+#pragma region Concrete Methods
+		/// <summary>
+		/// Enables or disables the Misc Settings menu.
```

Comment:
![uzira-confused](https://user-images.githubusercontent.com/50018358/123845460-e2a97480-d91c-11eb-8fce-64d4af466112.png)

Reply:
Our method comments generally use the class name rather than whatever you have here. I.e. `Enables or disabled the SettingsMiscGUI.`
Now I think about it, I think you've been kinda inconsistent with that in various places lol.